docs/mosviz/displayspectra.rst

```diff
@@ -2,7 +2,14 @@
 Display Spectra
 ***************
 
-Coming soon
+Spectra loaded by the user will be automatically displayed in the viewers, with 2d spectra
+going in the 2d spectrum viewer (top right section) and 1d spectra going in the 1d spectrum viewer
+(top right section, below the 2d spectrum viewer).
```

Comment:
```suggestion
populating the top right viewer, and 1d spectra populating the viewer 2nd from top on the right.```

"going in" rubs me the wrong way, and I think this can be a little more succinct while conveying the necessary information. I also agree with Patrick's suggestion above.